src/dagmc/dagmcmetadata.cpp

```diff
@@ -7,9 +7,12 @@ const char graveyard_str[] = "Graveyard";
 const char vacuum_str[] = "Vacuum";
 
 // constructor for metadata class
-dagmcMetaData::dagmcMetaData(moab::DagMC* dag_ptr, bool verbosity) {
+dagmcMetaData::dagmcMetaData(moab::DagMC* dag_ptr,
+                             bool verbosity,
+                             bool require_density_present) {
```

Comment:
I think modern C++ advice to use member initialization rather than assignment in constructors.

as you add a new member maybe it is a good place to start ?

Reply:
I had the same thought. There are a few other things I'd like to update as well. If it's alright I'll plan on submitting those in a separate PR as you suggested.

Reply:
sounds good for me
Again this is definitely **NOT** a blocking point.